Troubleshooting concurrency groups (#2503)
buildkite/docs: pages/pipelines/controlling_concurrency.md

Concurrency groups guarantee that jobs will be run in the order that they were created in. Jobs inherit the creation time of their parent. Parents of jobs can be either a build or a pipeline upload job. As pipeline uploads add more jobs to the build after it has started, the jobs that they add will inherit the creation time of the pipeline upload rather than the build.

>📘 Troubleshooting

> > When a build is blocked by a concurrency group, you can run this [GraphQL query](/docs/apis/graphql/cookbooks/jobs#get-all-jobs-in-a-particular-concurrency-group) that will list all the jobs that are in the queue for that particular concurrency group.

Reviewer: I think this is a bit clearer:

```suggestion
> When a build is blocked by a concurrency group, you can check which jobs are in the queue and their state using this [GraphQL query](/docs/apis/graphql/cookbooks/jobs#get-all-jobs-in-a-particular-concurrency-group).
```

Reviewer 2: I've already committed this suggestion @pzeballos ; I'm assuming this wording revision is OK from your perspective?
